brightwind/analyse/analyse.py

```diff
@@ -783,16 +783,19 @@ def time_continuity_gaps(data):
     """
     indexes = data.dropna(how='all').index
     resolution = tf._get_data_resolution(indexes)
-    resolution_days = resolution / pd.Timedelta('1 days')
+    # If the data resolution is `1 month` or `1 year`, then the resolution will 
```

Comment:
@BiancaMorandi could you please add a "be" at the end of this line? ".....resolution will **be**....."